### PR TITLE
Added fileList and urlList lambda functions

### PR DESCRIPTION
**Pull request by lakshya-20**
- Added `lambda_for_fileList` to get details of all files that are uploaded by a user.
- Added `lambda_for_urlList` to get details of all URLs corresponding to a file.

### CODE REVIEW

**Review by ayush0x00 — APPROVED** [review]
lgtm

**Review by danishjamal104 — COMMENTED** [review]
@lakshya-20 , where is the pagination implemented in lambda_for_fileList ??

File: lambdas/file_access_patterns/lambda_for_fileList/index.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            },
+            ExpressionAttributeValues:{
+                ':pk':`USER#${event.path.u_id}`,
+                ':sk':"FILE#"
+            },                        
+            Limit:10,
+        }
+        if(lastEvaluatedKey !== undefined) params.ExclusiveStartKey = lastEvaluatedKey;
+        const file_data = await dynamo.query(params).promise()        
+        file_data.LastEvaluatedKey = jsonToBase64(file_data.LastEvaluatedKey);
```
Comment (danishjamal104) [inline lambdas/file_access_patterns/lambda_for_fileList/index.js]: what if `file_data.LastEvaluatedKey` is undefined ??

Comment (lakshya-20) [reply lambdas/file_access_patterns/lambda_for_fileList/index.js]: `jsonToBase64` will return undefined in such a case.

Comment (danishjamal104) [reply lambdas/file_access_patterns/lambda_for_fileList/index.js]: so do you think its the right thing to return undefined for any attribute to frontend??

Comment (danishjamal104) [reply lambdas/file_access_patterns/lambda_for_fileList/index.js]: @ayush0x00 , your thought on this..

File: lambdas/file_access_patterns/lambda_for_fileList/index.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        return response(200,file_data);
+    }
+    catch(err){
+        return response(err.statusCode,err.message);
+    }
+
```
Comment (danishjamal104) [inline lambdas/file_access_patterns/lambda_for_fileList/index.js:53]: remove this

**Review by danishjamal104 — APPROVED** [review]

**Pull request merged**